renderer/components/editor/options/export-select.js

```diff
@@ -24,8 +24,12 @@ const Select = ({options}) => (
   </select>
 );
 
+Select.propTypes = {
+  options: PropTypes.array.isRequired
+};
+
 const ExportSelect = ({options = [], label}) => {
-  const hasOptions = options.length !== 0;
+  const hasOptions = options && options.length !== 0;
```

Comment:
This shouldn't be needed as `options` is marked as required in the proptypes. `options = []` is also wrong then as it will always be defined.